doc/design/overview.md

```diff
@@ -1,6 +1,8 @@
 # Loaders Design
 
-There are currently [three loader hooks](https://github.com/nodejs/node/tree/master/doc/api/esm.md#hooks):
+There are currently the following [loader hooks](https://github.com/nodejs/node/blob/master/doc/api/esm.md#hooks):
```

Comment:
If we're changing this link, let's avoid putting reference to the default branch name whose name may change in the future.
```suggestion
There are currently the following [loader hooks](https://github.com/nodejs/node/blob/HEAD/doc/api/esm.md#hooks):
```